foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -56,6 +56,10 @@ An important implication to notice here is that when you specify `flex-grow` or
 
 `flex-basis` simply sets the initial size of a flex item, so any sort of `flex-grow`ing or `flex-shrink`ing starts from that baseline size. The shorthand value defaults to `flex-basis: 0%`. The reason we had to change it to `auto` in the `flex-shrink` example is that with the basis set to `0`, those items would ignore the item's width, and everything would shrink evenly. Using `auto` as a flex-basis tells the item to check for a width declaration (`width: 250px`).
 
+### flex-auto
```

Comment:
**issue:** Just noting that it's actually `flex: auto`. The property is `flex` and the value for the property is `auto`. 

**suggestion:** Perhaps a good title for this section could be something like: What is flex: auto?

That might help distinguish it from the actual properties that are previously discussed as sections (`flex-grow`, `flex-shrink`, `flex-basis`).

To your comment about where to place this section, I believe we want it after the flex-basis section and I also think after the "Important note about flex-basis". 

To that end, I think we could either make this new section another note instead of a sub-header. So we could have the flex-basis section, then the note "Important note about flex-basis" and then another note like "What is flex: auto?" Or we could make it into its own sub-header section. We just want to make sure it is distinguished as a separate thing from the previous section properties. 

We could also take a look at the "Important note about flex-basis" note and probably remove the explanation on the equivalent value of `flex: auto` being `flex: 1 1 auto` since that would be covered in this new section. We could even signify that the learner will learn more about this in the next part with a rewrite of that final line like this: "You could also use the predefined value `flex: auto`, which you will see next."

Reply:
Hey! My bad for seeing this so late, I don't recieve the github notifications for odd reasons. Let me know your thoughts on the current phrasing in the recent push and hopefully I cleared the confusion on that one. If it's a bit wordy I still have the remaining drafts that we could consider 


- When `flex: auto` is defined it is equivalent to the values of `flex-grow: 1 flex-shrink: 1 flex-basis: auto (your option)
- When `flex: auto` is specified it is equivalent to the values of `flex-grow: 1 flex-shrink: 1 flex-basis: auto` or `flex: 1 1 auto`
- When `flex: auto` is defined as a flex parameter it is equivalent to the values of `flex-grow: 1 flex-shrink: 1 flex-basis: auto

Reply:
Hey do let me know if the placement is good around here, or if we'd rather it be a subheading on its own. I think it's great where its at personally